### ISSUE TITLE
Fix accordion header levels on Burial Benefits detail pages

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12183]
## Description
As a short term solution, because of a specific Veteran having an issue on certain pages, I would like to resolve the accordion header violations on the Burials benefits detail pages using the entity ID option.

### Additional Context
There are 3 Benefits detail pages within the Burials and Memorials section that need adjustments to their accordion headers:

- [Eligibility](https://www.va.gov/burials-memorials/eligibility/)
  - There are 2 accordion groups on this page. Both sets are currently using the default H4 heading
  - First set needs to change from the default H4 to H3
  - Second set should remain using the default H4

![image](https://user-images.githubusercontent.com/106678594/210886181-4e6b23e8-c6f4-4dea-ba73-bc4de78b2fcc.png)

- [Burial in a Private Cemetery](https://www.va.gov/burials-memorials/eligibility/burial-in-private-cemetery/)
  - There is one accordion group on this page.  Currently these are using the default H4 level but they should be using H5. **Can this be done with the entity ID list solution?** 
 
![image](https://user-images.githubusercontent.com/106678594/210886052-c515d400-7198-40e2-953d-32c91bebf447.png)

- [Veterans headstones, markers, and medallions](https://www.va.gov/burials-memorials/memorial-items/headstones-markers-medallions/)
- There is one accordion group on this page.  Currently these are using the default H4 level but they should be using H3. 
 
![image](https://user-images.githubusercontent.com/106678594/210886527-6c9fd41c-1600-4fb7-887e-ccc93d1890cf.png)

## Acceptance Criteria
- [ ] Update first accordion group on Eligibility page to H3
- [ ] Confirm second accordion group on Eligibility page remains H4
- [ ] Update accordion group on Burial in a Private Cemetery page to H5 (if possible)
- [ ] Update accordion group on Veterans headstones, markers, and medallions page to H3
- [ ] Accessibility Lead Review

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [x] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by laflannery** [comment]
**For future reference**

Prior to the updates being made this is the pa11y-ci error report that I generated for the burials section detail pages

- Axe runner
- WCAG2AA standard
- Looking for only heading violations on these pages
2 issues found (some of the issues we corrected above were not skipped levels but they were using the same level so still need to be corrected which is why they did not get reported here)
![image](https://user-images.githubusercontent.com/106678594/214914406-f5a9fd41-00dc-4454-97e3-b45bbd3779ff.png)


I will run this same report again later this afternoon when the fixes go live and post results

**Comment by laflannery** [comment]
Heading level fixes are now on production, I reran the pa11y-ci scan and 0 violations are reported:
![image](https://user-images.githubusercontent.com/106678594/215098368-bd3c153d-b926-4a0b-a7f5-d23580993f34.png)
